core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -901,4 +901,16 @@ <h2>Suggestions to review</h2>
       content: none;
     }
   }
+  @media(max-width: 316px){
+    .layout-row{
```

Comment:
Scope this down to the component. It would affect the whole website if you use this.